lib/course.rb

```diff
@@ -1 +1,24 @@
+class Course
+  attr_reader :name,
+              :capacity,
+              :students
 
+  def initialize(course_name, capacity)
+    @name = course_name
+    @capacity = capacity
+    @students = []
+    @full = false
+  end
+
+  def full?
+    if @students.count >= 2
+      @full = true
+    else
+      false
+    end
+  end
```

Comment:
Line 14 should not be hard-coded. What if the course `@capacity` is 3, or 20, etc?
Also, you could simplify the syntax further: 
```suggestion
  def full?
    @students.count >= @capacity
  end
```